webservices/resources/committees.py

```diff
@@ -103,35 +103,50 @@
 }
 
 
+def filter_year(model, query, years):
+    return query.filter(
+        or_(*[
+            and_(
+                or_(
+                    extract('year', model.last_file_date) >= year,
+                    model.last_file_date == None,
+                ),
+                extract('year', model.first_file_date) <= year,
+            )
+            for year in years
+        ])
+    )  # noqa
+
+
 class CommitteeList(Resource):
     parser = reqparse.RequestParser()
     parser.add_argument('q', type=str, help='Text to search all fields for')
-    parser.add_argument('committee_id', type=str, help="Committee's FEC ID")
-    parser.add_argument('candidate_id', type=str, help="Candidate's FEC ID")
-    parser.add_argument('state', type=str, help='Two digit U.S. State committee is registered in')
+    parser.add_argument('committee_id', type=str, action='append', help="Committee's FEC ID")
+    parser.add_argument('candidate_id', type=str, action='append', help="Candidate's FEC ID")
+    parser.add_argument('state', type=str, action='append', help='Two digit U.S. State committee is registered in')
```

Comment:
Updated. Once the API docs are ready, I think we should have a thorough review of the help text.